apps/condo/domains/document/access/DocumentCategory.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Generated by `createschema document.DocumentCategory 'name:Text; order:Integer;'`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function canReadDocumentCategories ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin) return {}
+
+    return {}
+}
+
+async function canManageDocumentCategories ({ authentication: { item: user }, operation }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    if (user.isSupport && (operation === 'create' || operation === 'update')) {
+        return true
+    }
```

Comment:
What is this for?
Without translations added by developers, it is useless

Reply:
After the developer adds a translation, the support user should be able to add a category

Reply:
I don't think so. It's important for such thing to have static IDs across multiple stands, so it can be safely hard-coded in 3rd party apps.

That's the thing we used in Ticket, Billing and other categories. We add it manually via migration.